src/components/View.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/prop-types, react/prefer-stateless-function */
```

Comment:
- [ ] Please don't disable linters for prop-types as requested by previous reviewer. PropTypes are very important to react components because they prevent to you from passing wrong props or wrong dataTypes. :blush: